<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QSlider" name="verticalSliderR">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::NoTicks</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxR">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QSlider" name="verticalSliderG">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxG">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSlider" name="verticalSliderB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxB">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="labelRGB">
        <property name="font">
         <font>
          <pointsize>35</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>RGB</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="text">
          <string>RadioButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QListView" name="listColor"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>verticalSliderR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>86</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderR</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxG</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderG</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSliderB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxB</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSliderB</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
